src/test/scala/org/broadinstitute/dsde/agora/server/AgoraTestData.scala

```diff
@@ -110,7 +110,7 @@ object AgoraTestData {
                                 |
                                 | """.stripMargin)
   val payloadReferencingExternalMethod = Option( """
-                                                   |import "methods://broad.wc.1"
+                                                   |import "http://broad.wc.1"
```

Comment:
this change gets us past the validation step that checks the scheme, but obviously we still fail because it does not resolve